hack/tools/tilt-prepare/main.go

```diff
@@ -413,6 +414,11 @@ func providerTask(path, out string, debug bool) taskFunction {
 			}
 		}
 
+		if err := prepareDeploymentForObservability(prefix, objs); err != nil {
```

Comment:
I added the modifications by default, as I did not know how I should pass the `deploy_observability` config option to `tilt-prepare`.

Reply:
I think that's fine. It doesn't hurt to add an additional port / replace the bind address if it was already there before.